test/test_subprocess_popen/test_from_import.py

```diff
@@ -83,3 +83,13 @@ def test_subfunction(number, **kwargs):
 
 def subfunction(name='World'):
     print(f"{name}!")
+
+
+@pytest.mark.unit
+@base_test_decorator
+@lug.run(image=BASE_TEST_IMAGE)
+def test_without_shell(number, **kwargs):
+    """Tests behavior when shell=True is not specified and a list is passed in as args."""
+    run_value = Popen(["/bin/ls", "-l"])
+    print("return code:", run_value.wait())
```

Comment:
If this fails to be re-routed, wouldn't this (and all the tests below) still work?

It seems like the tested functionality isn't whether this returns the same number its passed, but that `ls` is returning the paths that we expect.